backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/objcexport/ObjCExportCodeGenerator.kt

```diff
@@ -974,7 +978,8 @@ private fun ObjCExportCodeGenerator.createTypeAdapter(
             }
         }
         else -> {
-            // Nothing special.
+            if (descriptor.isNothing())
```

Comment:
This code adds `unit` class method to `Nothing` Objective-C wrapper class, and that method returns `Unit` instance.
What's the purpose of doing this?

Reply:
Maybe none, then, removed.